numbers.qmd

```diff
@@ -720,28 +719,24 @@ Finally, don't forget what you learned in @sec-sample-size: whenever creating nu
 
 ### Positions
 
-There's one final type of summary that's useful for numeric vectors, but also works with every other type of value: extracting a value at a specific position.
-You can do this with the base R `[` function, but we're not going to cover it in detail until @sec-subset-many, because it's a very powerful and general function.
-For now we'll introduce three specialized functions that you can use to extract values at a specified position: `first(x)`, `last(x)`, and `nth(x, n)`.
+There's one final type of summary that's useful for numeric vectors, but also works with every other type of value: extracting a value at a specific position: `first(x)`, `last(x)`, and `nth(x, n)`.
 
 For example, we can find the first and last departure for each day:
 
 ```{r}
 flights |> 
   group_by(year, month, day) |> 
   summarize(
-    first_dep = first(dep_time), 
-    fifth_dep = nth(dep_time, 5),
-    last_dep = last(dep_time)
+    first_dep = first(dep_time, na_rm = TRUE), 
+    fifth_dep = nth(dep_time, 5, na_rm = TRUE),
+    last_dep = last(dep_time, na_rm = TRUE)
   )
 ```
 
-(These functions currently lack an `na.rm` argument but will hopefully be fixed by the time you read this book: <https://github.com/tidyverse/dplyr/issues/6242>).
+(NB: Because dplyr functions use `_` to separate components of function and arguments names, these functions use `na_rm` instead of `na.rm`.)
```

Comment:
```suggestion
Note that because dplyr functions use `_` to separate components of function and arguments names, these functions use `na_rm` instead of `na.rm`.
```